### PR TITLE
Sync editor issues

### PR DESCRIPTION
**Pull request by jeswanke**
stolostron/backlog#21945
stolostron/backlog#21995
stolostron/backlog#22010

https://github.com/stolostron/backlog/issues/21611

### CODE REVIEW

File: frontend/src/components/SyncEditor/decorate.ts [hunk 1]
```diff
@@ -101,18 +125,29 @@ const addChangeDecorations = (
     decorations: any[]
 ) => {
     changes.forEach((chng) => {
-        const { $t, $a } = chng
+        const { $t, $a, $f } = chng
         const obj: any = get(change.mappings, $a)
         if (obj) {
+            //if (!isCustomEdit) {
```
Comment (KevinFCormier) [inline frontend/src/components/SyncEditor/decorate.ts:131]: @jeswanke Should probably remove this line?

Comment (jeswanke) [reply frontend/src/components/SyncEditor/decorate.ts:131]: ah, yeah--cleanup :)

**Pull request merged**